devel/llvm-morello-c18n/Makefile

```
@@ -0,0 +1,25 @@
+# llvm-morello
+
+LLVM_SUFFIX=	-morello
```

```suggestion
LLVM_SUFFIX=	-morello-c18n
```

Unless we cannot do that, it'd be better if we could install llvm-morello-c18n alongside llvm-morello.